packages/governance/src/types.js

```diff
@@ -583,8 +606,17 @@
  * @param {ParamSpecification} paramSpec
  * @param {ParamValue} proposedValue
  * @param {Installation} voteCounterInstallation
- * @param {bigint} deadline
- * @returns {ParamChangeVoteResult}
+ * @param {Timestamp} deadline
+ * @returns {ContractGovernanceVoteResult}
+ */
+
+/**
+ * @callback VoteOnApiInvocation
+ * @param {string} apiMethodName
+ * @param {[unknown]} methodParams
```

Comment:
brackets with contents indicate a tuple. I think this is an array (single type for all elements and unknown length).
```suggestion
 * @param {unknown[]} methodArgs
```